client/src/partials/templates/payslipEmployee.tmpl.html

```diff
@@ -0,0 +1,148 @@
+<div class="panel panel-primary">
+  <div ng-click="row.visible=!row.visible" class="panel-heading">
```

Comment:
This should be an `isVisible` attribute on the directive scope, since `employee.visible` doesn't really make a lot of sense.